packages/perseus/src/util/graphie.test.ts

```diff
@@ -1320,6 +1321,40 @@ describe("Graphie drawing tools", () => {
             expect(onSetDrawingAreaAvailable).toHaveBeenCalledWith(false);
         });
     });
+
+    describe("getMousePx", () => {
+        it.each([
+            [{left: 0, top: 0}, {pageX: 10, pageY: 10}, [10, 10]],
+            [{top: 10, left: 20}, {pageX: 40, pageY: 40}, [20, 30]],
```

Comment:
Is there a reason `left` and `top` are in the opposite order here? (same thing in the `it.each` on line 1343.)